apis/dispatch.php

```diff
@@ -77,6 +77,8 @@
 
 require_once("./../interface/globals.php");
 require_once("./../library/acl.inc");
+// Include our routes and init routes global
+require_once("./../_rest_routes.inc.php");
```

Comment:
Routes needs to be available regardless of authentication. If you need to ensure classes are loaded, do so in _rest_routes by running autoloader there.